src/rhizoscan/root/pipeline/arabidopsis.py

```diff
@@ -1,3 +1,6 @@
+from cv2 import erode as _erode
+from sklearn.cluster import KMeans as _Kmean
```

Comment:
The KMeans methods from scikit-learn it's more advanced, contrary to scipy, you can choose the number of time that the k-means algorithm will be run with different centroid seeds, and the filnal results will be the best output of consecutive runs in terms of inertia. More, the scikit methods return directly the input data labelized.

Since scikit-learn is a reference in python package for this kind of cluster function, fully supported and present any difficulty of installation on multi operating system (pip, conda, apt-get, whl), i preferred used this methods.